circle.yml

```diff
@@ -116,7 +116,18 @@ jobs:
                     ACTUAL_VERSION=$(docker compose run test-factory-all-included cypress -v)
                     TRIMMED_ACTUAL_VERSION=$(echo ${ACTUAL_VERSION} | sed -e 's/ Cypress binary version:.*$//g')
                     if [  "Cypress package version: ${CYPRESS_VERSION}" != "${TRIMMED_ACTUAL_VERSION}" ]; then
-                      echo "NodeVersion mismatch, Cypress package version: ${CYPRESS_VERSION} != ${TRIMMED_ACTUAL_VERSION}"
+                      echo "Version mismatch, Cypress package version: ${CYPRESS_VERSION} != ${TRIMMED_ACTUAL_VERSION}"
+                      exit 1;
+                    fi
+                    echo "Version ${ACTUAL_VERSION} confirmed"
+                  working_directory: factory/test-project
+            - run:
+                  name: check git version
+                  command: |
+                    ACTUAL_VERSION=$(docker compose run test-factory-all-included git --version)
+                    TRIMMED_ACTUAL_VERSION=$(echo ${ACTUAL_VERSION} | sed -e 's/ Cypress binary version:.*$//g')
```

Comment:
Is piping into `sed` needed? Looks like that would just apply to the Cypress version.

Reply:
oh, no, dead code

Reply:
removed in a7a6582